Provisioner/BuildProvisioner.php

```
@@ -78,7 +75,7 @@ public function onBuild(BuildEvent $event)
                 $polling['databases'] = $this->createSockDatabase($applicationEnvironment, $server);
             } catch (\Exception $ex) {
                 $this->taskService->addFailedLogMessage($this->task, 'Provisioning failed.');
-                $event->stopPropagation();
+                $this->task->setFailed();
```

Maybe we should throw an exception and let the caller mark the task as failed?

---

LoggedException?

---

yes